workflow_session/ingest.py

```diff
@@ -0,0 +1,112 @@
+import csv
+from workflow_session.pipeline import lab, subject, session
+
+
+def ingest_general(csvs, tables,
+                   skip_duplicates=True, verbose=True):
+    """
+    Inserts data from a series of csvs into their corresponding table:
+        e.g., ingest_general(['./lab_data.csv', './proj_data.csv'],
+                                 [lab.Lab(),lab.Project()]
+    ingest_general(csvs, tables, skip_duplicates=True)
+        :param csvs: list of relative paths to CSV files
+        :param tables: list of datajoint tables with ()
+        :param verbose: print number inserted (i.e., table length change)
+    """
+    for insert, table in zip(csvs, tables):
+        with open(insert, newline='') as f:
+            data = list(csv.DictReader(f, delimiter=','))
+        if verbose:
+            prev_len = len(table)
+        table.insert(data, skip_duplicates=skip_duplicates,
+                     # Ignore Extra because some CSVs feed/mult tables
+                     ignore_extra_fields=True)
+        if verbose:
+            insert_len = len(table) - prev_len     # report length change
+            print(f'\n---- Inserting {insert_len} entry(s) '
+                  + f'into {table.table_name} ----')
+
+    # Future enhancement: permit embedded lists
+    # Currently requires a csv to be listed multiple times
+    # if feeding multiple tables. Could instead take lists of lists.
```

Comment:
```suggestion
```

Reply:
I am inclined to make these GitHub Issues, but we can keep here if you think it is more appropriate.

Reply:
Sure thing - added issue #17